src/commands/Admin/reload.js

```diff
@@ -22,21 +23,39 @@ module.exports = class extends Command {
 					if (this.shard.id !== ${this.client.shard.id}) this.${piece.name}.loadAll().then(() => this.${piece.name}.loadAll());
 				`);
 			}
-			return message.sendMessage(`${message.language.get('COMMAND_RELOAD_ALL', piece)} (Took: ${timer.stop()})`);
+			return message.sendMessage(`${message.language.get('COMMAND_RELOAD_ALL', piece, timer.stop())}`);
```

Comment:
Remove the template literal, it's unneded, since message.language.get returns a string.